cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/KafkaRoller.java

```diff
@@ -126,6 +130,14 @@ public class KafkaRoller {
     private final Reconciliation reconciliation;
     private final boolean allowReconfiguration;
     private Admin allClient;
+    private static final String BROKER_STATE_REST_PATH = "/v1/broker-state";
+    private static final int BROKER_STATE_HTTPS_PORT = 8443;
+    private static final int BROKER_RECOVERY_STATE = 2;
+    private PemTrustOptions pto;
+    private PemKeyCertOptions pkco;
+    private static final String CLUSTER_CA_CERT_SECRET_KEY = "ca.crt";
+    private static final String CO_KEY_SECRET_KEY = "cluster-operator.key";
+    private static final String CO_KEY_CERT_SECRET_KEY = "cluster-operator.crt";
```

Comment:
Maybe the `private static final` variable can be defined up top with the other `static` variables?